lore/settings.py

```diff
@@ -353,5 +353,16 @@ def get_var(name, default):
     }
 }
 HAYSTACK_SIGNAL_PROCESSOR = 'haystack.signals.RealtimeSignalProcessor'
-ALLOW_CACHING = get_var("ALLOW_CACHING", get_var("ALLOW_CACHING", False))
+
 XANALYTICS_URL = get_var('XANALYTICS_URL', "")
+
+CACHES = {
+    "default": {
+        "BACKEND": get_var(
+            "CACHE_BACKEND", "django.core.cache.backends.locmem.LocMemCache")
```

Comment:
Yes, unfortunately. If you define anything and don't define default, it [blows up](https://github.com/django/django/blob/master/django/core/cache/__init__.py#L33).